src/middleware/io/can_interface.h

```diff
@@ -0,0 +1,183 @@
+// Copyright 2009-2018 Toby Schneider (http://gobysoft.org/index.wt/people/toby)
```

Comment:
Should I be up here instead?

Reply:
I currently use a script to update the copyright regularly (obviously I've fallen a bit behind as it's 2020). If you update that script too https://github.com/GobySoft/goby3/blob/3.0/scripts/update_copyright.sh to not overwrite your name then I think it would make sense.

Reply:
Would be nice to autogenerate this list of names from `git shortlog`, e.g. 

```
git shortlog -n -s  can_interface.h:

     2  Shawn Dooley
     1  Toby Schneider

```

Reply:
Actually I'll do this. Stand by!

Reply:
Updated the "./update_copyright.sh" script to be a bit smarter and list the authors in the commit history, ordered by the number of lines in `git blame`: https://github.com/GobySoft/goby3/pull/146

Reply:
Ok this should be set now. If you want to change your email on here, just update .mailmap and re-run `update_copyright.sh` in goby3/scripts.

Reply:
Wow. Quite Thorough. I think that is just fine.